docs/en/data_types/domains/domains.md

```diff
@@ -0,0 +1,24 @@
+# Domains
+
+Domains are special-purpose types, that add some extra features atop of existing 'base' type, leaving on-wire and on-disc format of underlying table intact. At the moment, Clickhouse does not support user-defined domains, and builtin domains are limited in functionality.
```

Comment:
* Click**H**ouse
* I'd remove "and builtin domains are limited in functionality"
* Why single quotes around base?